CMakeLists.txt

```diff
@@ -94,6 +94,12 @@ set (STBIMAGE_ROOT_DIR ${PROJECT_DEP_DIR/stb_image})
 target_include_directories(game PUBLIC ${PROJECT_DEP_DIR}/stb_image)
 ####################################################
 
+# json #############################################
+# Header only library for parsing json
+set (STBIMAGE_ROOT_DIR ${PROJECT_DEP_DIR/json})
```

Comment:
STBIMAGE_ROOT_DIR should be JSON_ROOT_DIR or something here (preferably the name ofthe library)